x-pack/plugins/security_solution/public/resolver/types.ts

```diff
@@ -699,13 +731,13 @@ export type PanelViewAndParameters =
          */
         nodeID: string;
         /**
-         * A value used for the `nodeEventsOfType` view. Used to associate this view with a parent `nodeEventsOfType` view.
-         * e.g. The user views the `nodeEventsOfType` and follows a link to the `eventDetail` view. The `eventDetail` view can
-         * use `eventType` to populate breadcrumbs and allow the user to return to the previous filter.
+         * Used to associate this view (via breadcrumbs) with a parent `nodeEventsInCategory` view.
+         * e.g. The user views the `nodeEventsInCategory` panel and follows a link to the `eventDetail` view. The `eventDetail` view can
+         * use `eventCategory` to populate breadcrumbs and allow the user to return to the previous view.
          *
-         * This cannot be inferred from the event itself, as an event may have any number of 'eventType's.
+         * This cannot be inferred from the event itself, as an event may have any number of eventCategories.
```

Comment:
❔ `as an event may have any number of eventCategories` is that true? Can it have 0?

Reply:
I think so. I'm basing this on the assumption that all fields in ES can have 0 or more values.